idaes/generic_models/properties/core/examples/tests/test_CO2_H2O_Ideal_VLE.py

```diff
@@ -0,0 +1,390 @@
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(default={"dynamic": False})
+        m.fs.properties = GenericParameterBlock(default=configuration)
+        m.fs.flash = Flash(default={"property_package": m.fs.properties})
+        
+        # Fix inlet stream state variables
+        m.fs.flash.inlet.flow_mol.fix(9.89433124673833) # mol/s
+        m.fs.flash.inlet.mole_frac_comp[0, 'CO2'].fix(0.13805801934749645)
+        m.fs.flash.inlet.mole_frac_comp[0, 'H2O'].fix(0.8619419806525035)
+        m.fs.flash.inlet.pressure.fix(183430) # Pa
+        m.fs.flash.inlet.temperature.fix(396.79057912844183) # K
+        
+        # Fix flash and its outlet conditions
+        m.fs.flash.deltaP.fix(0)
+        m.fs.flash.vap_outlet.temperature.fix(313.15)
+        
+        #Initialize the flash model
+        initial = m.fs.flash.initialize(outlvl=0)
+        
+        # Create a dictionary of expected solution for flash outlet temperature 
+        #sweep
+        temp_range = list(np.linspace(313,396))
+        
+        expected_vapor_frac = [0.14388,0.14445,0.14508,0.14576,0.1465,0.14731,
+                               0.14818,0.14913,0.15017,0.15129,0.15251,0.15384,
+                               0.15528,0.15685,0.15856,0.16042,0.16245,0.16467,
+                               0.16709,0.16974,0.17265,0.17584,0.17935,0.18323,
+                               0.18751,0.19226,0.19755,0.20346,0.21008,0.21755,
+                               0.22601,0.23565,0.24673,0.25956,0.27456,0.29229,
+                               0.31354,0.33942,0.37157,0.4125,0.46628,0.53993,
+                               0.64678,0.81547,1.0,1.0,1.0,1.0,1.0,1.0]
+        
+        expected_heat_duty = [-319572.91269,-318207.35249,-316825.31456,
+                              -315425.46452,-314006.35514,-312566.41317,
+                              -311103.92414,-309617.01485,-308103.633,
+                              -306561.52359,-304988.20138,-303380.91855,
+                              -301736.62677,-300051.9324,-298323.04327,
+                              -296545.70535,-294715.12686,-292825.88683,
+                              -290871.8244,-288845.90382,-286740.04885,
+                              -284544.93807,-282249.74995,-279841.84278,
+                              -277306.349,-274625.65625,-271778.73623,
+                              -268740.26687,-265479.46928,-261958.54546,
+                              -258130.54709,-253936.4179,-249300.81058,
+                              -244126.04073,-238283.13381,-231598.19109,
+                              -223830.94744,-214639.75318,-203521.76902,
+                              -189705.16711,-171941.46564,-148070.35054,
+                              -114001.22402,-60937.07508,-3219.88715,
+                              -2631.66029,-2043.09203,-1454.17752,-864.91582,
+                              -275.30623]
+        
+        outvals = zip(expected_vapor_frac,expected_heat_duty)
```

Comment:
Just as an observation (don't go a and change it), but I would have just created the `dict` of expected values directly. The reason being it would be a little more readable and maintainable for future, as you could easily cross reference  temperature with expected results. That said, the current form works so it is fine.